### DISCUSSION TITLE
How to make Grid list layout instead of Table

### DISCUSSION THREAD

**Discussion by maatata** [orchidsoftware/platform#1730] [Q&A]
Hi All

I need to make a screen which would list all the posts in a screen but instead of showing them in a table layout I would like to show them in Card layout.

So like a screen with 4 columns and each column would have multiple cards which each card relates to 1 post.
Each card will then have a post title, short description and image and some actions menu.

Also it needs pagination too on the page if it's more than the number of items which are supposed to show.

How would I be able to achieve such this layout please?

**Comment by maatata** [comment]
Any solution on this anyone?

**Reply by KrenZfer** [reply]
i think you need custom template or layout for this,
it will make more doable for that case
